src/services/platform.service.js

```diff
@@ -72,33 +72,38 @@ export const getBaseProjectEnvironment = (
   };
 };
 
-window.childProcess = childProcess;
-
 // HACK: With electron-builder, we're having some issues on mac finding Node.
 // This is because for some reason, the PATH is not updated properly :(
 // 'fix-path' is supposed to do this for us, but it doesn't work, for unknown
 // reasons.
 export const initializePath = () => {
-  childProcess.exec('which node', { env: window.process.env }, (_, version) => {
-    if (!version && isMac) {
-      // For users with a standard Node installation, node will be in
-      // /usr/local/bin
-      // For users using NVM, the path to Node will be added to `.bashrc`.
-      // Add both to the PATH.
-      try {
-        childProcess.exec(
-          'source ~/.bashrc && echo $PATH',
-          (err, updatedPath) => {
-            if (updatedPath) {
-              window.process.env.PATH = `/usr/local/bin:${updatedPath}`;
-            }
+  return new Promise((resolve, reject) => {
+    childProcess.exec(
+      'which node',
+      { env: window.process.env },
+      (_, version) => {
+        if (!version && isMac) {
+          // For users with a standard Node installation, node will be in
+          // /usr/local/bin
+          // For users using NVM, the path to Node will be added to `.bashrc`.
+          // Add both to the PATH.
+          try {
+            childProcess.exec(
+              'source ~/.bashrc && echo $PATH',
```

Comment:
Instead of assuming that people use bash, we should just launch a shell and get the path (it would be `~/.zshrc` for me for example).
```suggestion
              'echo $PATH',
              { shell: true, encoding: 'utf8' },
```

Reply:
The issue is that just `echo $PATH` doesn't find Node, since with NVM it's only added to the path in `.bashrc`, and for some reason, the built Electron app isn't loading it before starting.

This is specific to NVM though - I'm not sure how NVM installations work on Z shell.

But yeah, given that our target audience is beginners, I'm not _too_ worried about folks running NVM in zsh.

If you or anyone wants to dig into this problem and try and understand the root cause, that'd be awesome (this is definitely a hacky band-aid), but I've sunk too many hours into it now.